src-built-in/components/toolbar/components/AutoArrange.jsx

```diff
@@ -67,19 +67,29 @@ export default class AutoArrange extends React.Component {
 	}
 
 	render() {
-		const autoArrangedCss = this.state.isAutoArranged ? "auto-arranged" : "";
+		// For 4.0 auto-arrange is disabled -- it will be re-enabled in a future release after it is improved and hardened.
```

Comment:
Remove comments such as this. This repo is public and it is exposing things we may not want to.

Reply:
The comment documents the reality and explains the presence of the code, especially since disabling AA won't be viable for long.  But bottom line is I did remove it.